tox.ini

```diff
@@ -364,11 +364,14 @@ deps =
     -r dev-requirements.txt
     -r benchmarks/scripts/requirements.txt
     -r benchmarks/micro/requirements-compression-algorithms.txt
-    pytest-benchmark[histogram]
+    # need < 9.0.0 for python 2.7
+    py-cpuinfo==8.0.0
+    pytest-benchmark[histogram]==3.2.3; python_version < '3.1'
```

Comment:
I guess it was meant 3.11

Reply:
Good catch - I believe the reason it still works is because this requirement is duplicated in two places (requirements file, tox file) so the correct version still gets installed under Python 3.11.